stw_potsdam/views.py

```diff
@@ -11,6 +11,8 @@
 from stw_potsdam import feed
 from stw_potsdam.config import read_canteen_config
 from stw_potsdam.canteen_api import MenuParams, download_menu
+from stw_potsdam.swp_webspeiseplan_api import SWP_Webspeiseplan_API
+from stw_potsdam.swp_webspeiseplan_parser import SWP_Webspeiseplan_Parser
```

Comment:
At this time, the `/canteens` endpoint still grabs the canteens from the config file.

You already did the hard part and parsed the list of canteens from the API.
Using the information from the API to return the list of canteens would put a capstone to your work and would make the users more happy, because the parser added "Abholung / Pick up: Wohnheim Stahnsdorfer Straße" to the list of canteens, for example.
What do you think?

Reply:
We still need to retrieve openMensaIDs for a specific canteen, i don't know how they are genereated, or who maintains that. Can you provide a list with such ID for the mentioned canteens?

Reply:
What is an openMensaID in this context? I do not think the parser maintains that.
So we can delay this step, because I do not think "Abholung / Pick up: Wohnheim Stahnsdorfer Straße" is registered  with openMensa as of today, because the pickup location is not on the map on https://openmensa.org/#15/52.3923/13.1266.

Let's write a ticket for the guy with openMensa credentials, eventually he can add canteens, and later we can unlock the new canteens in the parser.

Are you OK with postponing this step to a later PR?

Reply:
#22